cwf/gateway/services/uesim/servicers/iperf_server_restarter.go

```diff
@@ -0,0 +1,86 @@
+package servicers
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/golang/glog"
+	"golang.org/x/crypto/ssh"
+	"time"
+)
+
+// restartIperfServer is a best efort function to restart iperf server, Will try to
+// access the iperf server through ssh using the address and port defined in uesim.
+// If ssh is not accessible it will just print error.
+
```

Comment:
note this is an experimental code. So we use hardcoded user and passwords. 

If this works, we will update this into a config file

BTW, those user/passwd are default, no secret is revealed